dist/forum-design-system.css

```diff
@@ -1109,6 +1109,15 @@ a.underlined-link, .underlined-links a {
 .profile-photo-overlap > .profile-photo-small:not(:first-child) {
   margin-left: -4px; }
 
+.profile-photo-overlap *:not(:first-child) .profile-photo-large {
+  margin-left: -12px; }
+
+.profile-photo-overlap *:not(:first-child) .profile-photo-medium {
+  margin-left: -8px; }
+
+.profile-photo-overlap *:not(:first-child) .profile-photo-small {
+  margin-left: -4px; }
+
```

Comment:
If we're gonna allow this, maybe we should remove the direct-child (`>`) versions?

Reply:
Hmm, I thought it would be okay to allow it to work with both?

Reply:
What I mean is that your change is a strictly better version of the rule for our given needs.
No reason to keep the old rule around.
Not a huge deal though, if you don't want to run another release.